src/Xabe.FFmpeg/Conversion/FFmpegWrapper.cs

```diff
@@ -141,6 +152,18 @@ private void ProcessOutputData(DataReceivedEventArgs e, string args, int process
             CalculateTime(e, args, processId);
         }
 
+        private void ProcessVideoData(Process process)
+        {
+            BinaryReader br = new BinaryReader(process.StandardOutput.BaseStream);
```

Comment:
You added possibility to change pipe output but here is always listening on StandardOutput, is it correct?

Reply:
We are limited to only using standard output for video data because both stdin and stderr are used already (for progress update??).

I figured being able to pipe the output to stdin and stderr may be useful in future since FFmpeg does support it, but it is not directly useful for piping the video data.

Reply:
Ok, great work :)
I'm going to merge it.